.gitignore

```diff
@@ -12,3 +12,4 @@ _ReSharper*/
 *.patch
 /packages
 *.ide/
+/.vs/config/applicationhost.config
```

Comment:
:bulb: This should be just the following:

```
.vs/
```

It also replaces the previous `*.ide/` exclusion for everything except some preview releases of Visual Studio 2015.